### PR TITLE
CHE-7581 Improved keycloak initialization

### PR DESCRIPTION
**Pull request by vitaliy-guliy**
### What does this PR do?
Improves keycloak initialization.

### What issues does this PR fix or reference?
https://github.com/eclipse/che/issues/7581

### CODE REVIEW

**Review by vparfonov — APPROVED** [review]

File: ide/che-ide-gwt-app/src/main/resources/org/eclipse/che/ide/public/IDE.html [hunk 1]
```diff
@@ -56,6 +56,81 @@
 
         var Loader = new function() {
 
+            /*
+             * Load keycloak settings
+             */
+            this.loadKeycloakSettings = function() {
+                var msg = "Cannot load keycloak settings";
```
Comment (mshaposhnik) [inline ide/che-ide-gwt-app/src/main/resources/org/eclipse/che/ide/public/IDE.html]: Please add explanation in message that this is normal sutuation for single-user mode.

**Review by mshaposhnik — APPROVED** [review]

File: ide/che-ide-gwt-app/src/main/resources/org/eclipse/che/ide/public/IDE.html (hunk 1 above)
Comment (vitaliy-guliy) [reply ide/che-ide-gwt-app/src/main/resources/org/eclipse/che/ide/public/IDE.html]: Done

**Comment by vitaliy-guliy** [comment]
ci-build

**Comment by vitaliy-guliy** [comment]
ci-test

**Review by ashumilova — APPROVED** [review]

**Review by musienko-maxim — APPROVED** [review]

**Pull request merged**